### PR TITLE
Docs issues

### PR DESCRIPTION
**Pull request by rkuchan**
Fixes issues:

* #762: Update the `Why aren't data dropped after I've altered a retention policy?` FAQ for 1.0.2 behavior.
* #522: Add documentation for `fill()` behavior with no data.
* #476: Add FAQ on mixing aggregates and non-aggregates.
* #772: Remove mentions of the `default` RP and replace with `autogen` in the 1.0 Kapacitor docs.
* #680: Update description of auth errors to include `403`.

### CODE REVIEW

**Comment by rkuchan** [comment]
@beckettsean, ready for review.

File: content/influxdb/v1.0/troubleshooting/frequently-asked-questions.md [hunk 1]
```diff
@@ -127,24 +128,16 @@ Check your retention policy's shard group duration with the
...
 
-First, by default, InfluxDB checks to enforce a RP every 30 minutes.
+By default, InfluxDB checks to enforce a RP every 30 minutes.
 You may need to wait for the next RP check for InfluxDB to drop data that are
 outside the RP's new `DURATION` setting.
 The 30 minute interval is
 [configurable](/influxdb/v1.0/administration/config/#check-interval-30m0s).
 
-Second,
-changing an RP's `DURATION` [may cause](#what-is-the-relationship-between-shard-group-durations-and-retention-policies)
```
Comment (beckettsean) [inline content/influxdb/v1.0/troubleshooting/frequently-asked-questions.md]: This info is still true, isn't it? We don't subdivide already-made shards if someone changes the RP.

Is this the one you moved to its own question?

Comment (rkuchan) [inline content/influxdb/v1.0/troubleshooting/frequently-asked-questions.md]: I was only thinking about `DURATION` changes no longer automatically resetting the `SHARD DURATION` (https://github.com/influxdata/influxdb/issues/7150) which led to unexpected data retention. Now it'll return an error if `DURATION` is less than the shard duration.

Having some trouble thinking through this but will the inability to subdivide shards still cause unexpected data retention?

**Comment by beckettsean** [comment]
last update looks good, Regan, thanks!

**Pull request merged**